src/index.css

```diff
@@ -131,6 +135,10 @@ code {
   padding: 0 !important;
 }
 
+.p-6 {
+  padding: 6px;
+}
+
```

Comment:
create new CSS module and classes for respective HTML elements in BC create edit page

Reply:
Yes, both options are possible, for now I would like to keep it as generic stylesheet until NCL-8445 and NCL-8443 are started.